cpp/harvest/nacjd.cc

```diff
@@ -343,9 +355,14 @@ void ParseJSONAndWriteMARC(MARC::Writer * const title_writer) {
 
 
 int Main(int argc, char *argv[]) {
-    if (argc != 3)
+    if (argc < 3 or argc > 4)
         Usage();
 
+    int count_rows(9000);
+    if (argv[3]) {
```

Comment:
Argc is allowed to be 3, and in this case accessing argv[3] will cause an error. Instead of `if (argv[3]) {` you should check for `if (argc == 4) {` instead.